styles/bsky/catppuccin.user.css

```diff
@@ -599,6 +599,17 @@
     circle[fill="hsl(211, 28%, 24.8%)"][r="1.5"] {
       fill: @surface1 !important;
     }
+
+    /* "save changes" button in edit profile */
+    div[style*="background-image: linear-gradient(94.0544deg, rgb(90, 113, 250), rgb(0, 133, 255))"] {
+      background: @accent-color !important;
+    }
+    div[style*="background-image: linear-gradient(94.0544deg, rgb(90, 113, 250), rgb(0, 133, 255))"] div {
+      color: @crust !important;
+    }
+    button[style*="transition-duration: 0s"] {
+      background: none !important;
+    }
```

Comment:
this can be matched with `[data-testid="editProfileSaveBtn"]` which is already done further up in the style